docs/samples/Microsoft.ML.AutoML.Samples/Microsoft.ML.AutoML.Samples.csproj

```diff
@@ -3,21 +3,27 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <NoWarn>CS0649</NoWarn>
```

Comment:
Whats this warning for?

Reply:
It's for uninitialized field, I can remove that after removing titanic example

Reply:
### Resolved